addons/osfstorage/models.py

```diff
@@ -523,8 +523,9 @@ def before_fork(self, node, user):
     def after_fork(self, node, fork, user, save=True):
         clone = self.clone()
         clone.owner = fork
-        clone.user_settings = self.user_settings
-        clone.region_id = self.region_id
+        user_settings = user.get_addon('osfstorage')
+        clone.user_settings = user_settings
```

Comment:
Hmm! Yeah, I think you're right - it does seem likely from this that all forks have the original creators user settings... 

I spent a bit trying to get the right query to find the number of things this would apply to, this is obvs not right cause target isn't doable but it'd be sooooomething maybe like... 

`NodeSettings.objects.filter(user_settings=F('root_node__target__creator__addons_osfstorage_user_settings')).count()`

I can see this being enough of a seprate issue that it could be in a new ticket tho...